tests/everest/conftest.py

```diff
@@ -255,25 +273,7 @@ def run_config(test_data_case: str):
 
 @pytest.fixture
 def min_config():
```

Comment:
Suggest to move this fixture to the top, right after the definiton of MIN_CONFIG.